include/rootfs.mk

```
@@ -87,6 +87,11 @@ define prepare_rootfs
 			fi; \
 		done || true \
 	)
+	gawk -i inplace \
```

```suggestion
	awk -i inplace \
```

According to the https://github.com/openwrt/openwrt/blob/main/include/prereq-build.mk#L136-L138 I would say, that this needs to be `awk`.

---

Changed to `awk`.